pootle/static/js/stats.js

```diff
@@ -305,10 +306,14 @@ const stats = {
 
     $(dirtySelector).toggleClass('dirty', !!data.is_dirty);
     if (!!data.is_dirty) {
-      this.dirtyBackoff = Math.pow(2, this.retries);
-      this.updateDirtyBackoffCounter();
-      $('.js-stats-refresh').show();
-      this.dirtyBackoffId = setInterval(() => this.updateDirty(), 1000);
+      if (this.isStatsBannerAutoRefreshEnabled) {
+        this.dirtyBackoff = Math.pow(2, this.retries);
+        this.updateDirtyBackoffCounter();
+        $('.js-stats-refresh').show();
+        this.dirtyBackoffId = setInterval(() => this.updateDirty(), 1000);
```

Comment:
If this is not necessary to be run after previous line, then previous line can be moved outside of `if`.